src/styles/mixins.scss

```diff
@@ -93,7 +95,9 @@ $spacer: 1.875rem; // 30px
 	$hover-background-color: $default-hover-background-color,
 
 	$font-size: $default-font-size,
-	$open-border-color: $default-open-border-color
+	$open-border-color: $default-open-border-color,
+
```

Comment:
Perfect, this is exactly what I was thinking! Non-breaking, but is in the right place 💯